src/plugins/mer/merbuildconfiguration.cpp

```diff
@@ -50,11 +56,25 @@ namespace Internal {
 
 MerBuildConfiguration::MerBuildConfiguration(Target *target, Core::Id id)
     : QmakeBuildConfiguration(target, id)
-    , m_useExtraParserArguments(false)
+    , m_useExtraParserArguments(MerSettings::isImportQmakeVariablesEnabled())
```

Comment:
WRT my comment in an earlier commit, MerSettings::isImportQmakeVariablesEnabled() can be used directly, no need to cache it here, no need for toMap/fromMap.

Reply:
Done.